Sources/BraveShared/Extensions/SceneExtensions.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+
+extension UIView {
```

Comment:
This probably belongs in `BraveUI` not `BraveShared` since its UI related extensions

Reply:
Fix at https://github.com/brave/brave-ios/pull/6284/commits/5118b6f9a517fd2916b9fd7d6ee626efe322926d